src/post/mod.rs

```diff
@@ -56,21 +59,26 @@ pub enum StorageKey {
     LabelToPostsV2,
     AuthorToAuthorPosts,
     AuthorPosts(CryptoHash),
+    Proposals,
+    LabelToProposals,
+    AuthorProposals,
```

Comment:
Oh, you have to add new keys to the end of the list, since the order of the keys is used to assign borsh-serialized value. NEAR SDK collections operate on the contract "local storage" and these keys are used as a prefix, so since there is already data stored with the prefixes 12-13 (Communities, AddOns), we don't want to break it by moving Communities to have prefix 15 since there is already data stored under prefix 12.